config/config.go

```diff
@@ -47,6 +47,7 @@ var (
 		"agent.flow.sflow.bind_address":        {"sflow.bind_address"},
 		"agent.flow.sflow.port_min":            {"sflow.port_min"},
 		"agent.flow.sflow.port_max":            {"sflow.port_max"},
+		"agent.topology.blockdev.lsblk_path":   {"lsblk.path"},
```

Comment:
relocationMap is for legacy configuraion (renamed or moved entries)

Reply:
ok - will remove